swagger_to_sdk_config.json

```diff
@@ -6,14 +6,19 @@
       "csharp": "",
       "reflect-api-versions": "",
       "license-header": "MICROSOFT_MIT_NO_VERSION",
-      "sdkrel:csharp-sdks-folder": "./sdk",
-      "use": "@microsoft.azure/autorest.csharp@2.3.82"
+      "use": "@microsoft.azure/autorest.csharp@2.3.84",
```

Comment:
Is this change related?

Reply:
Reverted. @PhoenixHe-msft did this change. He will create a seperate PR if anythings needs to be updated in configs.